lib/mobility/backends/active_record/key_value.rb

```diff
@@ -165,6 +165,10 @@ def visit_default(_)
         association_attributes = (instance_variable_get(:"@#{attrs_method_name}") || []) + attributes
         instance_variable_set(:"@#{attrs_method_name}", association_attributes)
 
+        translation_classes_method_name = :"__#{association_name}_translation_classes"
+        association_translation_classes = (instance_variable_get(:"@#{translation_classes_method_name}") || []) + Mobility::Backends::ActiveRecord::KeyValue::Translation.descendants
+        instance_variable_set(:"@#{translation_classes_method_name}", association_translation_classes)
```

Comment:
I'd really prefer to avoid setting new instance variables in the model class. I know it's done in other places but it's not a very good pattern, and I think there are better ways we can achieve the same thing. I'll comment on #525.